lang/sl/lang.php

```diff
@@ -0,0 +1,59 @@
+<?php return [
+    'plugin'             => [
```

Comment:
@BlazOrazem Please remove all the extra spacing between the first string and the arrow - there should only be one space to the left and right of the arrow.
```suggestion
    'plugin' => [
```
Please also do the same for all other lines below.

Reply:
@bennothommo It's fixed.